api/scripts/wazuh-apid.py

```diff
@@ -210,14 +210,13 @@ def set_logging(log_path=f'{API_LOG_PATH}.log', foreground_mode=False, debug_mod
     plain_log = 'plain' in api_conf['logs']['format']
     json_log = 'json' in api_conf['logs']['format']
 
-    log_path = f'{API_LOG_PATH}.log'
-    foreground_mode = args.foreground
-
+    if plain_log:
+        set_logging(log_path=f'{API_LOG_PATH}.log', debug_mode=api_conf['logs']['level'],
+                    foreground_mode=args.foreground)
     if json_log:
-        log_path = f'{API_LOG_PATH}.json'
-        foreground_mode = args.foreground and not plain_log
+        set_logging(log_path=f'{API_LOG_PATH}.json', debug_mode=api_conf['logs']['level'],
+                    foreground_mode=args.foreground and not plain_log)
 
-    set_logging(log_path=log_path, debug_mode=api_conf['logs']['level'], foreground_mode=foreground_mode)
 
```

Comment:
There should be only one blank line between logical sections of the same function ([PEP8](https://peps.python.org/pep-0008/#blank-lines)). Two blank lines can be used to, for example, separate different functions, but this is not the case.

```suggestion
```